esp/eclwatch/ws_XSLT/dropzonefile.xslt

```diff
@@ -196,11 +197,11 @@
             function getSelectedDropZone()
             {
               machineDropDown = document.forms['DropZoneForm'].machine;
-              if ((machineDropDown.selectedIndex < 1) && (machineDropDown.length > 1) && (currentNetAddress != ""))
+              if ((machineDropDown.selectedIndex < 1) && (machineDropDown.length > 1) && (currentDropZone != "/"))
```

Comment:
Just a name. The 'machine' is the dropdown box in the legacy ECLWatch page. Each item inside the box contains machine(ServerList/server) + dropzone name.